js/test/root/teststrings.js

```diff
@@ -3432,6 +3432,26 @@ module.exports.teststrings = {
         test.equal(str.formatChoice(5), "Default items");
         test.done();
     },
+    testStringFormatChoiceCharClasses_yo_NG: function(test) {
+        test.expect(2);
+        var str = new IString("0#There are no items.|one#There items are one|few#The items are few|many#The items are many|#Default items");
```

Comment:
The item is one

Reply:
All value is updated including this test case.